shared/.env.sample

```diff
@@ -5,3 +5,4 @@ FIREBASE_AUTH_EMULATOR_HOST=127.0.0.1:9099
 
 STRIPE_SECRET_KEY="sk_test_XXXXXXXXXXXXXXXXXXXXXXXX"
 DATABASE_URL="postgresql://social-income:social-income@localhost:5432/social-income"
+EXCHANGE_RATES_IMPORT_API="dqIi92ch4OaQIrpbp36flxxuUY3b4rKG"
```

Comment:
Use a simpler placeholder value so people don’t think it’s the actual key used in staging or production.

And if the staging and production values aren’t already in 1Password, please add them.